src/main/java/org/folio/rest/impl/UsersAPI.java

```diff
@@ -399,7 +398,7 @@ public void deleteUsersByUserId(String userId, String lang,
         Criteria idCrit = new Criteria();
         idCrit.addField(USER_ID_FIELD);
         idCrit.setOperation("=");
-        idCrit.setValue(userId);
+        idCrit.setVal(userId);
```

Comment:
Can you replace the complete method by a PgUtil.deleteById invokation so that this duplicated code gets removed?